app/models/concerns/foreman/thread_session.rb

```diff
@@ -69,10 +69,15 @@ def current=(o)
             raise(ArgumentError, "Unable to set current User, expected class '#{self}', got #{o.inspect}")
           end
 
-          if o.is_a?(User)
-            Rails.logger.info "Current user: #{o.login} (#{o.admin? ? 'administrator' : 'regular user'})"
+          Rails.logger.debug?
```

Comment:
This should be written as 
ˋˋˋ
Rails.logger.debug do 
  ... this happens only in debug log level
end
```